src/inttest/java/com/faforever/api/featuredmods/FeaturedModsControllerTest.java

```diff
@@ -0,0 +1,26 @@
+package com.faforever.api.featuredmods;
+
+import com.faforever.api.AbstractIntegrationTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.jdbc.Sql;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = "classpath:sql/truncateTables.sql")
+@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = "classpath:sql/prepDefaultData.sql")
+@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = "classpath:sql/prepFeaturedMods.sql")
+public class FeaturedModsControllerTest extends AbstractIntegrationTest {
+
+  @Test
+  public void featuredModFileUrlCorrect() throws Exception {
+    mockMvc.perform(get("/featuredMods/0/files/latest")
+        .with(getOAuthTokenWithActiveUser(NO_SCOPE, NO_AUTHORITIES)))
```

Comment:
LOBBY_SCOPE instead of NO_SCOPE?

Reply:
Yep still need to fix and add tests this was just a first pass